quantum/audio/driver_chibios_pwm.h

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2020 Jack Humbert
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#if !defined(AUDIO_PWM_TIMER)
+// NOTE: Timer2 seems to be used otherwise in QMK, otherwise we could default to A5 (= TIM2_CH1, with PWMD2 and alternate-function(1))
+#    define AUDIO_PWM_TIMER 1
+#endif
+
+#if !defined(AUDIO_PWM_TIMERCHANNEL)
+// NOTE: sticking to the STM data-sheet numbering: TIMxCH1 to TIMxCH4
+// default: STM32F303CC PA8+TIM1_CH1 -> 1
+#    define AUDIO_PWM_TIMERCHANNEL 1
+#endif
+
+#if !defined(AUDIO_PWM_PINALTERNATE_FUNCTION)
```

Comment:
```suggestion
#if !defined(AUDIO_PWM_PAL_MODE)
```

Reply:
I would avoid shortening the define to this, since chibios already uses PAL as Pin-Abstraction-Layer

Reply:
except we use this notation in existing drivers, and we want to avoid introducing more fragmentation.

Reply:
the only similar notation is one occurrence in the pwm-backlight - but ok, aligning to that.

Reply:
`*_PAL_MODE` is used for spi/i2c/ws2812/serial(half duplex) drivers not just backlight.